src/routes.js

```diff
@@ -1,9 +1,15 @@
 const { Router } = require("express");
+const UserController = require("./controllers/UserController");
 
 const routes = new Router();
 
 routes.get("/", (req, res) => {
   res.json({ message: "Hello World" });
 });
 
+routes.get("/users", (req, res) => UserController.listAll(req, res));
```

Comment:
Não é necessária a arrow function aqui já que o método utilizado possui os mesmos parâmetros da arrow function.

```suggestion
routes.get("/users", UserController.listAll);
```